llvm/lib/Support/RISCVISAInfo.cpp

```diff
@@ -1062,79 +1059,82 @@ static const char *ImpliedExtsZvl65536b[] = {"zvl32768b"};
 static const char *ImpliedExtsZvl8192b[] = {"zvl4096b"};
 
 struct ImpliedExtsEntry {
-  StringLiteral Name;
+  const char *Name;
```

Comment:
Does this have compile time impact? Now we have to do a strlen call every time we search this table.

Reply:
I changed this because StringLiteral didn't have a constexpr `<` operator. But I took another stab at this, and I think we can actually make the StringLiteral std::string_view operator constexpr.